interface/reports/daily_summary_report.php

```diff
@@ -0,0 +1,409 @@
+    <head>
+        <?php html_header_show(); ?>
+        <link rel="stylesheet" href="<?php echo $css_header; ?>" type="text/css">
+        <script type="text/javascript" src="../../library/textformat.js"></script>
+        <script type="text/javascript" src="../../library/js/jquery.1.3.2.js"></script>
+        <script type="text/javascript">
+
+
+            var mypcc = '<?php echo $GLOBALS['phone_country_code'] ?>';
+
+            function submitForm() {
+                var fromDate = $("#form_from_date").val();
+                var toDate = $("#form_to_date").val();
+
+                if (fromDate === '') {
+                    alert("Please select From date");
+                    return false;
+                }
+                if (toDate === '') {
+                    alert("Please select To date");
+                    return false;
+                }
+                if (Date.parse(fromDate) > Date.parse(toDate)) {
+                    alert("From date should be less than To date");
+                    return false;
+                }
+                else {
+                    $("#form_refresh").attr("value", "true");
+                    $("#report_form").submit();
+                }
+            }
+        </script>
+
+    </head>
+
+    <body class="body_top">
+
+        <span class='title'><?php echo xlt('Daily'); ?> - <?php echo xlt('Summary') . ' ' . xlt('Report'); ?></span>
+        <!-- start of search parameters --> 
+        <form method='post' name='report_form' id='report_form' action='' onsubmit='return top.restoreSession()'>
+            <div id="report_parameters">
+                <table class="tableonly">
+                    <tr>
+                        <td width='745px'>
+                            <div style='float: left'>
+                                <table class='text'>
+                                    <tr>
+                                        <td class='label'><?php echo xlt('Facility'); ?>:</td>
+                                        <td><?php dropdown_facility($selectedFacility, 'form_facility', false); ?></td>				
+                                        <td class='label'><?php echo xlt('From'); ?>:</td>
+                                        <td>
+                                            <input type='text' name='form_from_date' id="form_from_date"
+                                                   size='10' value='<?php echo attr($from_date) ?>'
+                                                   onkeyup='datekeyup(this, mypcc)' onblur='dateblur(this, mypcc)'
+                                                   title='yyyy-mm-dd'> <img src='../pic/show_calendar.gif'
+                                                   align='absbottom' width='24' height='22' id='img_from_date'
+                                                   border='0' alt='[?]' style='cursor: pointer'
+                                                   title='<?php echo xlt('Click here to choose a date'); ?>'>
```

Comment:
since this is within a html attribute, use xla instead:
``````
title='<?php echo xla('Click here to choose a date'); ?>'>